### PR TITLE
[Android] added buttonmap for XBOX 360 controller

### PR DESCRIPTION
**Pull request by da-anda**
The first commit is adding the buttonmaps for the XBOX 360 controller for Android (for some reason the SHIELD uses a different ID for the controller than my TV, which is the reason why there are two buttonmaps for the 360 controller).

The second commit is adjusting the exiting button maps to work again after the change xbmc/xbmc#14959

### CODE REVIEW

**Comment by da-anda** [comment]
@garbear I think this PR is now good to go. I was unable to test the SHIELD v1.4 controller (don't have a new SHIELD) and also not the virtual controller (no idea how to test it).

File: peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml [hunk 1]
```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" ?>
 <buttonmap>
-    <device name="OUYA Game Controller" provider="android" vid="2836" pid="0001" buttoncount="18" axiscount="6">
+    <device name="Xbox 360 Wireless Receiver" provider="android" vid="045E" pid="02A1" buttoncount="23" axiscount="6">
```
Comment (garbear) [inline peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml]: Is this change intended?

Comment (da-anda) [reply peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml]: wtf Looks like a GIT rename in combination with a file change went wrong when committing the changes :(

Comment (da-anda) [reply peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml]: ok, just checked the individual commits and there it's fine (the ouya git rename still got lost somehow though). Seems to be github confusing things in the "Files changed" view

Comment (Rechi) [reply peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml]: This isn't the fault of GitHub, it's just how git works.

**Comment by garbear** [comment]
Let's get this in for testing. @da-anda you committed the original OUYA buttonmap, is it supposed to be missing buttons or was that because they were missing in the android driver you updated?

File: peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml (hunk 1 above)
Comment (da-anda) [reply peripheral.joystick/resources/buttonmaps/xml/android/Xbox_360_Wireless_Receiver_v045E_p02A1_23b_6a.xml]: @Rechi - if you check the individual commits here on Github then you see that what Garbear is commenting on is not part of any of the changes there.

**Comment by da-anda** [comment]
the Ouya controller was missing button because of xbmc/xbmc#14959 - so I created a test build of that PR and created a new buttonmap for the Ouya, the SHIELD 1.3 controller and added new ones for the XBOX 360 controller.

**Comment by garbear** [comment]
Cool, so this is ready to go?

**Comment by da-anda** [comment]
IMO yes, I see nothing wrong in this PR. And as said, the XBOX, SHIELD v1.3 and Ouya button maps are made "from scratch" using an appropriate test build, so these should be fine. Due to lack of a newer SHIELD I was not able to test SHIELD v1.4, but it should be fine as well.

**Comment by garbear** [comment]
Good enough for me :) Thanks!!!
